traditional_ML_algorithms/linear_regression/Salary_Data.csv

```diff
@@ -0,0 +1,31 @@
+YearsExperience,Salary
```

Comment:
As LinearRegression Already exists in the repo, please use the existing CSV file for training and visualization purposes. Delete this file and try to train your model on [Revenue.csv](https://github.com/Math-behind-AI/ScratchAI/blob/main/traditional_ML_algorithms/SimpleLinearRegression/data.csv) This will help in comparing the algorithms by different authors.